### PR TITLE
Add new roundend sound

### PR DESCRIPTION
**Pull request by Majick1**
## Changelog
:cl:
added new roundend sound
/:cl:

## About The Pull Request

New Sseth voiceline on @GuyonBroadway s copypasta, had to PR since nobody would just slip it in the folder

## Why It's Good For The Game

It's not bad

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
You sure I'm responsible for this and not KBB?

Like what was the source of this? Was I drunk again?

**Comment by Majick1** [comment]
[https://cdn.discordapp.com/attachments/276455180852264960/577274999996678155/azcqa2J.png](url)

Was a while back, had to wait for SSeth to reply and asked a few times like a month ago before having to PR it

**Comment by GuyonBroadway** [comment]
Yer link is 404ing

**Comment by Majick1** [comment]
GitHub being dumb, copypasting works

**Comment by ghost** [comment]
needs to be added to the ticker.dm or whatever

**Comment by Majick1** [comment]
Ok fixed

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -642,7 +642,8 @@ SUBSYSTEM_DEF(ticker)
 		'sound/roundend/its_only_game.ogg',
 		'sound/roundend/yeehaw.ogg',
 		'sound/roundend/disappointed.ogg',
-		'sound/roundend/scrunglartiy.ogg'\
+		'sound/roundend/scrunglartiy.ogg'
```
Comment (ghost) [inline code/controllers/subsystem/ticker.dm]: comma

**Comment by Majick1** [comment]
period

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -642,7 +642,7 @@ SUBSYSTEM_DEF(ticker)
 		'sound/roundend/its_only_game.ogg',
 		'sound/roundend/yeehaw.ogg',
 		'sound/roundend/disappointed.ogg',
-		'sound/roundend/scrunglartiy.ogg'
+		'sound/roundend/scrunglartiy.ogg',
```
Comment (ghost) [inline code/controllers/subsystem/ticker.dm]: wait a second THIS IS THE WRONG `ticker.dm`

**Review by ghost — DISMISSED** [review]
it compiles. so w/e

**Comment by Majick1** [comment]
removed rogue /

**Comment by ghost** [comment]
@Majick1 it doesn't compile without that, stupid.

**Review by ghost — DISMISSED** [review]
don't remove the `\` this time.

**Review by ghost — APPROVED** [review]
dumb honkbot

**Comment by AnthonyJNgo** [comment]
G2g, yeet

**Pull request merged**